run.py

```diff
@@ -140,6 +144,25 @@ def generate_app(
         allow_headers=["*"],
     )
 
+    @app.middleware("http")
+    async def block_origin_middleware(request: Request, call_next):
+        isValidOrigin: bool = False
+        if "Origin" not in request.headers:
+            isValidOrigin = True
```

Comment:
この条件は初見で理由がわからない人もいるかもとちょっとだけ思いました。
こちらもなにかコメントを書くのはどうでしょう。
```suggestion
        if "Origin" not in request.headers:  # GETリクエストの一部やブラウザを介さないリクエスト
            isValidOrigin = True
```

Reply:
コメントを付与するのはとても良いと思いましたが、この付近のロジックを確認する人であれば CORS の概要は知っていると思いますので、`# ブラウザの CORS を使用しないリクエスト`  などのほうが解りやすいかもと思いました！